### ISSUE TITLE
Update to support-library and play-services

### ISSUE THREAD

**Issue by stefanmedack** [googlecast/CastCompanionLibrary-android#350]
Could you please update the support library to the lastest version (25.2.0)? The play services will probably need an update too (10.2.0).

Currently lint is complaining that `customtabs` and `mediarouter-v7` use a different support-library than the latest one. I overcame this problem at the moment by including them manually again, but I'm afraid it could lead to some unpredictable bugs.

**Comment by naddaf** [comment]
If you add the higher versions of those dependencies to your own project, wouldn't the library then use them automatically?

**Comment by stefanmedack** [comment]
Yes, this is the part where I'm afraid of unpredictable Bugs. Since CCL is developed against support library 25.0.0 and maybe some changes from 25.2.0 break behavior this library is developed against.

It would just be nice to have this sorted out in the near future. For now, it builds and seems to work perfectly fine. Thank you

**Comment by naddaf** [comment]
My general thoughts on that is the following: support libs should be backward compatible. If, at some point, a new version of library causes an issue for CCL, I will update CCL to address that and bring it on the latest version of support library. Otherwise, I'll let it ride on its own.

**Comment by tomislavr** [comment]
@stefanmedack Use something like this to get rid of warning:

```
configurations.all {
    resolutionStrategy.force "com.google.android.gms:play-services-cast:${googleServicesVersion}" // cast.companion
    resolutionStrategy.force "com.android.support:mediarouter-v7:${supportServicesVersion}" // cast.companion
}
```

**Comment by stefanmedack** [comment]
@naddaf for play services 11.0.0 the cast companion library seems to be broken. I can select a device for casting, but nothing happens after that anymore. Can you reproduce this as well and maybe consider updating now? 
Thank you and kind regards
